src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.spec.ts

```diff
@@ -84,7 +84,9 @@ describe('SkyAgGridDataManagerAdapterDirective', () => {
       node: rowNode
     } as RowSelectedEvent;
 
-    dataState.selectedIds = ['1'];
+    dataState.selectedIds = [ '3', '1' ];
```

Comment:
Yup. Adding ids to the test data uncovered this. We weren't seeing one of the data points in this array because that data had no `id` even though it was set to `selected`. Adding the `id` caused it to show up in the data and the test started failing. This was correct behavior and I just had to tweak the test for it.